src/utils/validators.js

```diff
@@ -13,8 +13,9 @@ const isPositiveNumber = function ( str ) {
 
 
 /** Returns true if a string represents a positive integer */
+/** @todo Change name to 'isWholeNumber'. */
 const isPositiveWholeNumber = function ( str ) {
-  return /^[0-9]*$/.test(str);
+  return str !== '' && /^[0-9]*$/.test(str);
```

Comment:
Could use `/^\d+$/.test(str)`

Reply:
That would work too, but I find this more readable unless there's a particular advantage to `\d`.

Reply:
No, there isn't. I probably should have left that out, as I was mostly suggesting using `+` instead of `*` to avoid `str !== ''`.

Reply:
Oh, got it. Yeah, totally missed the `+`. That would probably be a better choice, though I did like how explicit `str !=== ''` is. Another for the list, though.